invisible_cities/cities/base_cities.py

```diff
@@ -422,17 +414,20 @@ def __init__(self, **kwds):
         self.rebin          = conf.rebin
         self.reco_algorithm = find_algorithm(conf.reco_algorithm)
 
-    def rebin_s2(self, S2, Si):
-        if self.rebin <= 1:
-            return S2, Si
-
-        S2_rebin = {}
-        Si_rebin = {}
-        for peak in S2:
-            t, e, sipms = cpf.rebin_S2(S2[peak][0], S2[peak][1], Si[peak], self.rebin)
-            S2_rebin[peak] = Peak(t, e)
-            Si_rebin[peak] = sipms
-        return S2_rebin, Si_rebin
+    #TODO ALEX SHOULD REVISE
+
+    # def rebin_s2(self, s2d, sid):
+    #     """rebins s2d and sid dictionaries"""
+    #     if self.rebin <= 1:
+    #         return s2d, sid
+    #
+    #     S2_rebin = {}
+    #     Si_rebin = {}
+    #     for peak in S2:
+    #         t, e, sipms = cpf.rebin_S2(S2[peak][0], S2[peak][1], Si[peak], self.rebin)
+    #         S2_rebin[peak] = Peak(t, e)
+    #         Si_rebin[peak] = sipms
+    #     return S2_rebin, Si_rebin
```

Comment:
I think we can delete all of this. This code calls a function, `cpf.rebin_S2`, that has not existed in nextic/master for months. `cpf.rebin_S2` seems to be a precursor to the old version of `rebin_waveform` that I recently changed.